test/edu-benefits/pages/applicantInformation.unit.spec.jsx

```diff
@@ -142,12 +148,16 @@ describe('Edu 1990e applicantInformation', () => {
 
 describe('Edu 5490 applicantInformation', () => {
   // Make sure these match the fields listed in the 5490
-  const { schema, uiSchema } = applicantInformation(fullSchema5490, [
-    'relativeFullName',
-    'relativeSocialSecurityNumber',
-    'relativeDateOfBirth',
-    'gender'
-  ]);
+  const { schema, uiSchema } = applicantInformation(fullSchema5490, {
```

Comment:
Yeah, I think we need to overhaul how we test shared pages (make sure they are actually unit test, not moving toward integration tests, but test the functionality we know is used by those forms)